src/sequences/Formula.ts

```diff
@@ -5,7 +5,83 @@ import {MathFormula} from '@/shared/math'
 import type {GenericParamDescription} from '@/shared/Paramable'
 import {ParamType} from '@/shared/ParamType'
 
+/** md
+# Formula sequence
+[<img
+  src="../../assets/img/Formula/Beatty.png"
+  style="margin-left: 1em; margin-right: 0.5em; max-width: 696px"
+/>](../assets/img/Formula/Beatty.png)
+
+The entries of this sequence are given by a mathematical formula that you
+enter in its parameters tab; for example `(n+1)n/2` would generate the
+triangular numbers. The formula (unless you want it to be a constant, i.e.,
+have all of its entries be the same) should mention the variable `n`. To
+generate the entry of this sequence at index 15 (say), Numberscope will
+substitute the value 15 in for `n` in the formula, and then evaluate the
+resulting expression. So to continue the above example, the entry at index
+15 of the Formula sequence given by `(n+1)n/2` will be `16*15/2`, or 120.
+
+All familiar arithmetic operations may be used in the formula:
+
+`+, -, *, /`
+:   addition, subtraction, multiplication, division. Note that as in typical
+    mathematical notation, the `*` for multiplication may be omitted when
+    the terms being multiplied can clearly be detected by the formula parser.
+    So `2n` is the same as `2*n`, but you can't write `ncos(n)` for `n*cos(n)`
+    because the parser doesn't know whether you perhaps meant the function
+    name to be `ncos`.
+
+`^`
+:   exponentiation, e.g. `2^3` will produce 8
+
+`%`
+:   remainder, e.g. `18 % 5` will produce 3 because that's the remainder
+    when dividing 18 by 5
```

Comment:
is this mathematical remainder (modulo), or javascript remainder (differs on negative values)?